scripts/jenkins/qa/cloud-mkphyscloud-qa-scenario-1a.yaml

```diff
@@ -137,8 +138,8 @@
           wget --no-check-certificate -Oqa_crowbarsetup.sh "$runner_url"
           wget --no-check-certificate -Oscenario.yml "$scenario_url"
 
-          sed -i -e "s,##shared_nfs_for_database##,10.162.26.129:/var/$cloud/ha-database," scenario.yml
-          sed -i -e "s,##shared_nfs_for_rabbitmq##,10.162.26.129:/var/$cloud/ha-rabbitmq," scenario.yml
+          sed -i -e "s,##shared_nfs_for_database##,10.162.31.220:/var/$cloud/ha-database," scenario.yml
```

Comment:
I guess the IPs are specific for qa1? makes sense to parametrize it so can be used for others? (sorry if the question is uninformed)